Makefile

```diff
@@ -65,3 +65,7 @@ src.test-coverage:
 check-govendor:
 	$(info check govendor)
 	@[ "`which $(GO_VENDOR)`" != "" ] || (echo "$(GO_VENDOR) is missing"; false)
+
+## dockerfiles/ ########################################
+docker.build:
+	docker build -t="sdnvortex/cv-server:latest" ./dockerfiles/
```

Comment:
use the `dockerfiles/dockerfile` as the dependency of target `docker.build`
Please rebase master and check the build won't fail.

By the way, use the variable for tag, such as
```
--tag $(DOCKER_REGISTRY)/$(PROJECT):$(IMAGE_TAG) \
--file $< .
```

Reply:
Remove `ARG BRAND=aurora`